### DISCUSSION TITLE
Can anyone help with a chat interface like Gradio on top of this and like conver it to api as well to use in other applications

### DISCUSSION THREAD

**Discussion by tushaar9027** [imartinez/privateGPT#642] [General]
Can anyone help with a chat interface like Gradio on top of this and like conver it to api as well to use in other applications

**Comment by pseudotensor** [comment]
If you want to stick to privateGPT, there's some UI PRs you can try.  If you want both UI + client for docs + LLMs, you can try h2oGPT: https://github.com/h2oai/h2ogpt .
